applications/sandbox/CMakeLists.txt

```diff
@@ -1,11 +1,19 @@
 # Sandbox depends on applications and graphics in this case
 
-if (${RYTHE_MODULE_APPLICATION} AND ${RYTHE_MODULE_GRAPHICS})
+if (${RYTHE_BUILD_APPLICATIONS} AND ${RYTHE_MODULE_GRAPHICS})
```

Comment:
This seems incorrect, I think you confused what this if statement was for - When this was written, sandbox depended on the Application module, RYTHE_BUILD_APPLICATIONS isn't that. If sandbox still depends on the application module, we should be checking if it's enabled.

You shouldn't have to check for RYTHE_BUILD_APPLICATIONS here, applications aren't enabled at all if RYTHE_BUILD_APPLICATIONS is disabled, meaning this code wouldn't run in the first place

Reply:
It could be interesting to explore a more explicit & generalized dependency system for applications, maybe they could report their dependencies and then the applications cmake could check if they can be run

Reply:
No the reason this happened was because i was testing rendering without the application module. it's temporary and it should be:
`if (${RYTHE_BUILD_APPLICATIONS} AND ${RYTHE_MODULE_APPLICATION} AND ${RYTHE_MODULE_GRAPHICS})`